css/styles.css

```diff
@@ -78,20 +96,53 @@ header .logo {
 }
 
 .container img {
-    float: left;
-    margin: 20px 0 0 0;
 }
 
 .container p {
     background-color: #9C4D13;
     color: #fff;
-    padding: 5px 0 5px 10px;
-    width: 580px;
 }
 
 /***** ----- Footer ----- *****/
 
 footer {
     background-color: #C6621A;
+    color: #fff;
     margin-top: 20px;
+}
+
+.column {
+    float: left;
+    width: 25%;
+}
+
+.column:last-child {
+    width: 50%;
+}
+
+.column h3 {
+    border-bottom: 1px solid #fff;
+    margin: 15px 10px 15px 0;
+    padding-right: 70px;
+}
+
+.column p {
+    background-color: #C6621A;
```

Comment:
This shouldn't be here. :octopus:  We are not working in the content yet so it would be safe to remove the `.container p` from the content section and it will solve your problem We are not working in the content yet so it would be safe to remove the `.container p` from the content section and it will solve your problem.